pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ValidationMetrics.java

```diff
@@ -184,6 +184,17 @@ public void updateTotalDocumentsGauge(final String resource, final long document
   }
 
   /**
+   *
+   * @param resource The resource for which the guage is updated
+   * @param partitionCount Number of kafka partitions that do not have any segment in CONSUMING state.
+   */
+  public void updateNumNonConsumingPartitions(final String resource, final int partitionCount) {
```

Comment:
I'd rename this to something updateNonConsumingPartitionMetric, right now it's not clear it's updating metrics (or what it's updating)